Command/TailCursorDoctrineODMCommand.php

```diff
@@ -24,12 +24,13 @@
  */
 class TailCursorDoctrineODMCommand extends Command implements ContainerAwareInterface
 {
+    /** @var string */
+    protected static $defaultName = 'doctrine:mongodb:tail-cursor';
     use ContainerAwareTrait;
```

Comment:
```suggestion
    use ContainerAwareTrait;
    
    /** @var string */
    protected static $defaultName = 'doctrine:mongodb:tail-cursor';
```

I am not sure why this was not caught by our coding standards, could you please change it as suggested? I hope the multi-line suggestion works :)

Reply:
fixed